doc/appendices/command-line/traffic_layout.en.rst

```diff
@@ -78,7 +78,7 @@ Subcommands
       Find the sandbox to remove in following order: 
 
             1. specified in --path as absolute or relative.
-            2. :envvar:`TS_RUNROOT`
+            2. ENV variable: $TS_RUNROOT
```

Comment:
Why change this? Define `TS_RUNROOT` elsewhere in the documentation as an `envvar`.

Reply:
It is not showing up as envvar because it is part of the code chunk in the doc.

Reply:
You have a numbered list in a code chunk? That doesn't seem well done.

Reply:
I'll reformat this to make it look better